docs/user_guide/activation-strategies.md

```diff
@@ -0,0 +1,88 @@
+
+The definition of an activation strategy lives in the Unleash API and can be created via the Unleash UI. The implementation of activation strategies lives in various client implementations.
+
+Unleash comes with a few common activation strategies. Some of them require the client to provide the [unleash-context](./unleash-context.md), which gives the necessary context for Unleash.
+
+## default
+
+It is the simplest activation strategy and basically means "active for everyone".
+
+## userWithId
+
+Active for users with a `userId` defined in the `userIds` list. Typically I want to enable a new feature only for myself in production before I enable it for everyone else. To achieve this, we can use the “UserWithIdStrategy”. This strategy allows you to specify a list of user IDs that you want to expose the new feature for. (A user id may, of course, be an email if that is more appropriate in your system.)
+
+**Parameters**
+
+- userIds - _List of user IDs you want the feature toggle to be enabled for_
+
+## flexibleRollout
+
+A flexible rollout strategy which combines all gradual rollout strategies in to a single strategy (and will in time replace them). This strategy have different options for how you want to handle the stickiness, and have sane default mode.
+
+**Parameters**
+
+- **stickiness** is used to define how we guarantee consistency for gradual rollout. The same userId and the same rollout percentage should give predictable results. Configuration that should be supported:
+  - **default** - Unleash chooses the first value present on the context in defined order userId, sessionId, random.
+  - **userId** - guaranteed to be sticky on userId. If userId not present the behavior would be false
+  - **sessionId** - guaranteed to be sticky on sessionId. If sessionId not present the behavior would be false.
+  - **random** - no stickiness guaranteed. For every isEnabled call it will yield a random true/false based on the selected rollout percentage.
+- **groupId** is used to ensure that different toggles will **hash differently** for the same user. The groupId defaults to _feature toggle name_, but the use can override it to _correlate rollout_ of multiple feature toggles.
+- **rollout** The percentage (0-100) you want to enable the feature toggle for.
+
+### Customize stickiness (beta)
+
+By enabling the stickiness option on a custom context field you can use it together with the flexible rollout strategy. This will guarantee a consistent behavior for specific values of this context field. PS! support for this feature currently being supported by the following SDKs:
```

Comment:
```suggestion
By enabling the stickiness option on a custom context field you can use it together with the flexible rollout strategy. This will guarantee a consistent behavior for specific values of this context field.
NB! this feature is currently only supported by the following SDKs:
```